docs/release-notes/release-notes-0.18.0.md

```diff
@@ -89,6 +89,15 @@
   the new payment status `Payment_INITIATED` should be used for payment-related
   RPCs. It's recommended to use it to provide granular controls over payments.
 
+* A [helper command (`lncli encryptdebugpackage`) for collecting and encrypting
+  useful debug information](https://github.com/lightningnetwork/lnd/pull/8188)
+  was added. This allows a user to collect the most relevant information about
+  their node with a single command and securely encrypt it to the public key of
+  a developer or support person. That way the person supporting the user with
+  their issue has an eas way to get all the information they usually require
+  without the user needing to publicly give away a lot of privacy-sensitive
```

Comment:
curious about how to obtain the public key - is it the same as the release signing key?

Reply:
We'll define one and then update the issue template and FAQ to mention the `lncli encryptdebugpackage` command and key to use when submitting info to us. And I'll probably create an internal tool that has the corresponding private key to decrypt the user files in a nice UI.